src/translations/en.json

```diff
@@ -3170,7 +3170,8 @@
                     "picker": "If the numeric value of an entity''s state (or attribute''s value) is above or below a given threshold.",
                     "above": "When {attribute, select, \n undefined {} \n other {{attribute} from }\n }{entity} is above {above}",
                     "below": "When {attribute, select, \n undefined {} \n other {{attribute} from }\n }{entity} is below {below}",
-                    "above-below": "When {attribute, select, \n undefined {} \n other {{attribute} from }\n }{entity} is above {above} and below {below}"
+                    "above-below": "When {attribute, select, \n undefined {} \n other {{attribute} from }\n }{entity} is above {above} and below {below}",
+                    "more_entities": " and {count} more entities"
```

Comment:
We do support multiple entities in other triggers and conditions as well, so I think we should take the poison pill in this case. 

You can use 
```import {
  formatListWithAnds
} from "../common/string/format-list";```
to get a intl formatted list of the entities. 

If you feel uncomfortable with these translations, let me know and I'll take care of it.

Reply:
Ok I have changed the keys as suggested. 

Do I (or someone) need to go on lokalize and wipe out all existing translations for these strings after this PR is merged? Or after we do a general release that includes this PR? 

Or do we just leave it along and let translators update it at their leisure. I thought I heard there was some process for this but I was never clear on it.

Reply:
They should update it at their leisure, I guess. 

When the translation has been changed in context, it's a good thing to introduce a new translation. The English one has been changed in context, but the other translations are still accepted and used and show up in Localize as unverified. 
When you no longer provided certain parameters, it's also a good thing to introduce a new translation. The other language translations get unverified in Localize, but they still expect a parameter to be passed in, which no longer is. 

In this case you added a parameter, that is when the translation has been updated already it will show correctly, when it is not it will just don't use the newly provided parameter and use the old translation. If you want to wipe keys, you should do it somewhere right after the PR but I'm not sure if you have permissions to do so. Any PR touching the translation file will trigger a CI to push the updated translations, so that's the time when translators will see the translation become unverified allowing them to update. If you wipe them on general release, you may wipe already translated translations.